cyclops-ctrl/internal/cluster/k8sclient/client.go

```diff
@@ -916,3 +935,7 @@ func isSecret(group, version, kind string) bool {
 func isCronJob(group, version, kind string) bool {
 	return group == "batch" && version == "v1" && kind == "CronJob"
 }
+
+func isRole(group, version, kind string) bool {
+	return group == "" && version == "v1" && kind == "Role"
```

Comment:
can you update the group here to `rbac.authorization.k8s.io`?

Reply:
Yeah sure